cmd/transport/options/options.go

```diff
@@ -0,0 +1,55 @@
+/*
+Copyright 2023 The KubeStellar Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package options
+
+import (
+	"github.com/spf13/pflag"
+)
+
+const (
+	defaultConcurrency = 4
+)
+
+type TransportOptions struct {
+	Concurrency         int
+	WdsKubeConfig       string
```

Comment:
The kubeconfig context seems to be an important variable too. There are other common `kubectl` flags that could be exposed. See https://github.com/kubestellar/kubestellar/blob/v0.14.0/cmd/placement-translator/main.go#L84-L88 for example.

Reply:
done.
I've added under `cmd` a new dir `client-options` which can be reused by the different controllers (e.g., placement controller needs it as well).
I then used it in the transport controller options+cmd.

Reply:
It's a little odd to put something under `cmd/` that is not a command. I myself would have put it in `pkg/`.
Not a blocker.